mailbox/cassandra/src/test/java/org/apache/james/mailbox/cassandra/CassandraMessageIdManagerQuotaTest.java

```diff
@@ -45,12 +46,12 @@ protected MaxQuotaManager createMaxQuotaManager() {
     }
 
     @Override
-    protected QuotaManager createQuotaManager(MaxQuotaManager maxQuotaManager, CurrentQuotaManager currentQuotaManager) {
+    protected QuotaManager createQuotaManager(MaxQuotaManager maxQuotaManager, StoreCurrentQuotaManager currentQuotaManager) {
```

Comment:
I don't agree with that change, you should not require a specific implementation for CurrentQuotaManager

Reply:
Comment outdated, already been fixed

Reply:
I mean `StoreQuotaManager` has been deleted in later commits